packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -79,23 +79,19 @@ class _DropdownMenuPainter extends CustomPainter {
 
 // Do not use the platform-specific default scroll configuration.
 // Dropdown menus should never overscroll or display an overscroll indicator.
-class _DropdownScrollConfigurationDelegate extends ScrollConfigurationDelegate {
-  const _DropdownScrollConfigurationDelegate(this._platform);
+class _DropdownScrollBehavior extends ScrollBehavior2 {
+  const _DropdownScrollBehavior(this.platform);
 
-  @override
-  TargetPlatform get platform => _platform;
-  final TargetPlatform _platform;
+  final TargetPlatform platform;
 
   @override
-  ExtentScrollBehavior createScrollBehavior() => new OverscrollWhenScrollableBehavior(platform: platform);
+  TargetPlatform getPlatform(BuildContext context) => platform;
 
   @override
-  Widget wrapScrollWidget(BuildContext context, Widget scrollWidget) {
-    return new ClampOverscrolls(edge: ScrollableEdge.both, child: scrollWidget);
-  }
+  Widget buildViewportChrome(BuildContext context, Widget child, AxisDirection axisDirection) => child;
 
   @override
-  bool updateShouldNotify(ScrollConfigurationDelegate old) => platform != old.platform;
+  ScrollPhysics getScrollPhysics(BuildContext context) => const ClampingScrollPhysics();
```

Comment:
The old code as explicit about not doing it.  I couldn't find any examples one way or another...